Makefile

```diff
@@ -44,7 +44,7 @@ test:
 
 # requires odoc
 doc:
-	dune build @doc
+	dune build @doc -profile=release
```

Comment:
I believe this needs`--profile` (two dashes)

Reply:
😳